tests/org.corpus_tools.hexatomic.it.tests/src/main/java/org/corpus_tools/hexatomic/it/tests/TestGridEditor.java

```diff
@@ -0,0 +1,352 @@
+package org.corpus_tools.hexatomic.it.tests;
+
+import static org.eclipse.swtbot.swt.finder.matchers.WidgetMatcherFactory.widgetOfType;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.corpus_tools.hexatomic.core.errors.ErrorService;
+import org.corpus_tools.hexatomic.core.handlers.OpenSaltProjectHandler;
+import org.corpus_tools.hexatomic.grid.style.StyleConfiguration;
+import org.eclipse.core.commands.ParameterizedCommand;
+import org.eclipse.e4.core.commands.ECommandService;
+import org.eclipse.e4.core.commands.EHandlerService;
+import org.eclipse.e4.core.contexts.IEclipseContext;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.jface.bindings.keys.KeyStroke;
+import org.eclipse.nebula.widgets.nattable.NatTable;
+import org.eclipse.swt.SWT;
+import org.eclipse.swtbot.e4.finder.widgets.SWTBotView;
+import org.eclipse.swtbot.e4.finder.widgets.SWTWorkbenchBot;
+import org.eclipse.swtbot.nebula.nattable.finder.SWTNatTableBot;
+import org.eclipse.swtbot.nebula.nattable.finder.widgets.SWTBotNatTable;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotCombo;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotTreeItem;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Integration tests for the grid editor.
+ * 
+ * @author Stephan Druskat (mail@sdruskat.net)
+ *
+ */
+@SuppressWarnings("restriction")
+public class TestGridEditor {
+
+  private SWTWorkbenchBot bot = new SWTWorkbenchBot(ContextHelper.getEclipseContext());
+
+  private URI exampleProjectUri;
+  private URI overlappingExampleProjectUri;
+  private URI twoDsExampleProjectUri;
+  private ECommandService commandService;
+  private EHandlerService handlerService;
+
+  private ErrorService errorService = new ErrorService();
+
+
+
+  @BeforeEach
+  void setup() {
```

Comment:
On Linux systems with Wayland, the default SWTBot strategy (`MockKeyboardStrategy`) does not work properly: <https://bugs.eclipse.org/bugs/show_bug.cgi?id=549680>. This causes the `testDropdownWithTwoTexts` test, which uses `pressShortcut()`, to fail on my system, when running it with `mvn integration-test. 

I would suggest to change the strategy to the `SWTKeyboardStrategy`. This is also tested for the other integration tests, to minimize the impact on possible execution related order changes. An alternative would be to enforce using XWayland for executing the tests, but this could proof to be tricky.
```suggestion
  void setup() {
    org.eclipse.swtbot.swt.finder.utils.SWTBotPreferences.KEYBOARD_STRATEGY =
        "org.eclipse.swtbot.swt.finder.keyboard.SWTKeyboardStrategy";
```